test/README

```diff
@@ -57,22 +57,22 @@ follows:
   .orientation[2] = n + 1
   .orientation[3] = n + 1
   .num_ranges     = n + 1
-  .ranges[i]      = i         (i = 0 ... n-1)
```

Comment:
Since `.num_ranges` is `n + 1` this needs to run to `n` and not `n - 1` as previously written.
This is also what is received in the client.